docs/messaging/server-integration.md

```diff
@@ -114,22 +114,24 @@ async function onUserPictureLiked(ownerId, userId, picture) {
   // Get the users details
   const user = admin.firestore().collection('users').doc(userId).get();
 
-  await admin.messaging().sendToDevice(
-    owner.tokens, // ['token_1', 'token_2', ...]
-    {
-      data: {
-        owner: JSON.stringify(owner),
-        user: JSON.stringify(user),
-        picture: JSON.stringify(picture),
-      },
+  await admin.messaging().sendEachForMulticast({
+    tokens: owner.tokens, // ['token_1', 'token_2', ...]
+    data: {
+      owner: JSON.stringify(owner),
+      user: JSON.stringify(user),
+      picture: JSON.stringify(picture),
     },
-    {
-      // Required for background/quit data-only messages on iOS
-      contentAvailable: true,
-      // Required for background/quit data-only messages on Android
-      priority: 'high',
+    apns: {
+      payload: {
+        aps: {
+          // Required for background/quit data-only messages on iOS
```

Comment:
```suggestion
          // Required for background/quit data-only messages on iOS
          // Note: iOS frequently will receive the message but decline to deliver it to your app.
          //           This is an Apple design choice to favor user battery life over data-only delivery 
          //           reliability. It is not under app control, though you may see the behavior in device logs.
```

Reply:
forgive me taking the opportunity to throw that note in there, it's a really high support load item in the repo though, I try to spread this knowledge anywhere people might see it...